### DISCUSSION TITLE
Panorma view all administrator and delete an administrator account

### DISCUSSION THREAD

**Discussion by avjay96** [PaloAltoNetworks/pan-os-python#422] [Q&A]
Hi everyone,
I am very new to panorama and have been trying to find out if there is a way to view all administrators that are given access to panorama and all how to delete an inactive panorama user account using pan-os-python or pan xml api?  Any help would really be appreciated

**Comment by shinmog** [comment]
```python
from panos.panorama import Panorama
from panos.device import Administrator

pano = Panorama(........)
admins = Administrator.refreshall(pano)

for admin in admins:
    # do stuff here
```
